htdocs/install/mysql/tables/llx_webhook_history_extrafields-webhook.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright (C) 2025		Alice Adminson				<myemail@mycompany.com>
```

Comment:
This table will be filled by system, not by end user, so the extrafields table to add custim field should be useless.
You can remove it.